helpwave/web
# fix: adopt react library for lucide svg icons

The Lucide react library icons have been implemented in this pull request, replacing all the existing SVG icons.

This resolves issue #205 regarding the adoption of the Lucide react library for SVG icons in the application.

Testing instructions:
1. To test, please ensure that all required packages specified in the package.json file are installed.
2. Then, thoroughly browse the web application to verify the icons are functioning properly.
